src/Tickets/Commerce/Gateways/PayPal/Client.php

```diff
@@ -434,9 +434,14 @@ public function create_order( array $units = [] ) {
 	 *
 	 * @return array|null
 	 */
-	public function capture_order( $order_id ) {
+	public function capture_order( $order_id, $order_data ) {
 		$query_args = [];
-		$body       = [];
+
+		// Todo @rafsuntaskin handle $data validation to avoid errors.
+		$body       = [
+			'payerID' => $order_data['payerID'],
```

Comment:
Need to make sure `$order_data['payerID']` exists before using it. I see in `src/Tickets/Commerce/Gateways/PayPal/Repositories/Order.php` that a blank array could be passed.